synpress.config.ts

```diff
@@ -36,9 +41,11 @@ export default defineConfig({
       safeAddDefaultLocalNetwork();
 
       // Set Cypress variables
+      config.env = { ...process.env };
```

Comment:
Does this help us when we are already assigning the env variables individually later?